### PR TITLE
Fix a few regexes

### PR DESCRIPTION
**Pull request by hreeder**
I've fixed regexes for a few functions

### CODE REVIEW

**Comment by hreeder** [comment]
With 4f49cf8 this PR will override #17 now

File: tigobot/scripts/pub.coffee [hunk 1]
```diff
@@ -2,6 +2,6 @@
 #   Pub!
 
 module.exports = (robot) ->
-  robot.hear /pub/i, (msg) ->
+  robot.hear /^pub\??$/i, (msg) ->
```
Comment (neanias) [inline tigobot/scripts/pub.coffee]: Could you change the regex to `/^pu+b\??$/i` to allow for such things as `<Norgg> puuuuub`

Comment (hreeder) [inline tigobot/scripts/pub.coffee]: That would be good yeah, will do when I'm back at a computer

**Comment by hreeder** [comment]
Updated this to support the following
Puuuuuub?
ppppuuubbbbbbbbbbbbbbbb
pub

**Comment by neanias** [comment]
:shipit:

**Pull request merged**